src/DayPicker/DayPicker.js

```diff
@@ -211,7 +211,7 @@ module.exports = kind(
 		if (str) {
 			return str;
 		}
-
+		this.selectedIndex.sort();
```

Comment:
Yes, indeed, I didn't insert this line because I thought there is no case that selectedIndex's order is changed. I also would like to know when and why selectedIndex is changed even rebuildSelectedIndices is called.